server/controller/loanUserController.js

```diff
@@ -52,37 +53,36 @@ export function getUserLoan(req, res) {
 }
 
 export function addNewLoan(req, res) {
-  let errorMessage = '';
-  let status = 400;
   const { userMail } = req;
   const { tenor, amount } = req.body;
-
-  if (!userMail) errorMessage = 'Please provide the user email';
-  else if (!tenor || isNaN(tenor) || tenor < 1) errorMessage = 'Tenor should be a strict positive number';
-  else if (!amount || isNaN(amount) || amount < 1) errorMessage = 'Amount should be a strict positive number';
-  else {
-    const [user] = getSingleUser(userMail);
-    const [currentLoans] = getPendingLoans(userMail);
-    if (!user) {
-      errorMessage = 'No user found for the given email';
-      status = 404;
-    } else if (currentLoans) {
-      errorMessage = 'You still have another pending loan';
-      status = 403;
-    } else {
-      // eslint-disable-next-line max-len
-      const newLoan = addUserLoan(new Loan.Loan(getLoanCount(), userMail, Number.parseInt(tenor, 10), Number.parseInt(amount, 10)));
-      res.status(201).send({
-        status: 201,
-        data: newLoan,
-      });
-    }
-  }
-
-  if (errorMessage) {
-    res.status(status).send({
-      status,
-      error: errorMessage,
+  const newLoan = new Loan.Loan(getLoanCount(), userMail, Number.parseInt(tenor, 10),
```

Comment:
`new Loan.Loan()` seems to be repetitive. Why not rename the method Loan to something more descriptive such as `addLoan` or `createLoan`